pkg/auth/is_jwt_token_expired_test.go

```diff
@@ -0,0 +1,71 @@
+package auth
+
+import (
+	"testing"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+	. "github.com/onsi/gomega"
+)
+
+const (
+	jwtKeyFile = "test/support/jwt_private_key.pem"
+	jwtCAFile  = "test/support/jwt_ca.pem"
+)
+
+func Test_IsJWTTokenExpired(t *testing.T) {
+	RegisterTestingT(t)
+
+	authHelper, err := NewAuthHelper(jwtKeyFile, jwtCAFile, "")
+	Expect(err).ToNot(HaveOccurred())
+
+	account, err := authHelper.NewAccount("testuser", "user", "user@email.com", "orgId")
+	Expect(err).ToNot(HaveOccurred())
+
+	expiredToken, err := authHelper.CreateSignedJWT(account, jwt.MapClaims{
+		"exp": time.Now().Add(-1 * time.Minute * time.Duration(5)).Unix(),
+	})
+	Expect(err).ToNot(HaveOccurred())
+
+	unExpiredToken, err := authHelper.CreateSignedJWT(account, jwt.MapClaims{
+		"exp": time.Now().Add(time.Minute * time.Duration(5)).Unix(),
+	})
+	Expect(err).ToNot(HaveOccurred())
+
+	type args struct {
+		accessToken string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "return true for expired accessToken",
+			args: args{
+				accessToken: expiredToken,
+			},
+			want: true,
+		},
+		{
+			name: "return false for unExpired accessToken",
+			args: args{
+				accessToken: unExpiredToken,
+			},
+			want: false,
+		},
+		{
+			name: "return false for invalid accessToken",
```

Comment:
we are expecting a a return value of "true" the text there needs to be updated to reflect the same.